com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/ProcessTypeJsonAdapter.java

```diff
@@ -8,6 +8,8 @@
 
 public class ProcessTypeJsonAdapter extends TypeAdapter<ProcessType> {
 
+    private static final String NULL_STRING = (String) null;
```

Comment:
Why not DEFAULT_NULL_VALUE or something similar?

Reply:
It is a String reference with a null value. There can be Boolean with null too. We use it to suggest which overloaded method to be called, so for readability the type of the reference should be present in the variable name. We can use STRING_NULL_VALUE.